src/Foundation/NSHost.cs

```diff
@@ -102,21 +110,26 @@ public static NSHost FromIPHostEntry (IPHostEntry hostEntry)
 		public IPHostEntry ToIPHostEntry ()
 		{
 			return new IPHostEntry {
-				HostName = Name,
+				HostName = Name!,
```

Comment:
For others looking at the PR, HostName can be null, as the default constructor shows, dotnet is missing the nullability there.